README.md

```diff
@@ -16,13 +16,14 @@ yarn add freedom-accounts-util
 ```
 const accounts = require('freedom-accounts-util');
 
-accounts.configure({base_url: '', path: '', client_id: '', client_secret: '', client_expiry: 300, server_expiry: 300, disable_caching: false});
+accounts.configure({base_url: '', path: '', client_id: '', client_secret: '', client_expiry: 300, server_expiry: 300, disable_caching: false, retry_count: ''});
```

Comment:
prettify? hard to read the fields.
```js
accounts.configure({
    base_url: '',
    path: '',
    client_id: '',
    client_secret: '',
    client_expiry: 300,
    server_expiry: 300,
    disable_caching: false,
    retry_count: ''}
);
```
Also, retry_count should be number, and should have default value of 3 I guess?

Reply:
And comment client_expiry, and server_expiry to be in seconds.

Reply:
Prettified :D Comment for client_expiry and server_expiry to be in seconds is already included in the `Notes` section of README